api/v1beta2/cluster_phase_types.go

```diff
@@ -49,6 +49,11 @@ const (
 
```

Comment:
Should bootstrap-provider.plantuml & machine-infra-provider.plantuml also be adjusted?

Reply:
I saw that bootsrap-provider and machine-phases was adjusted. I think we should also modify machine-infra-provider